### PR TITLE
[CMS-391] Modify docs to reflect WP Cron changes

### PR DESCRIPTION
**Pull request by jspellman814**
## Summary

[Cron for WordPress](https://pantheon.io/docs/wordpress-cron) - These documentation updates are based on the WP Cron changes being implemented in [#312](https://github.com/pantheon-systems/WordPress/pull/312) and [#9](https://github.com/pantheon-upstreams/wordpress-project/pull/9).

### Dependencies and Timing

- [x] PR [#312](https://github.com/pantheon-systems/WordPress/pull/312) is merged
- [x] PR [#9](https://github.com/pantheon-upstreams/wordpress-project/pull/9) is merged
- [x] After the next major/minor version of WordPress has been merged into the two repos above.

**Release**:
- [ ] After the dependencies above have been merged

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-7229](https://pr-7229--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `52` | `54` | `2` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `98` | `98` | `0` |

Lighthouse Audit Reports:

- [`pr-7229` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/35483/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/35483/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-7229--pantheon-docs.my.pantheonfrontend.website/docs)

File: source/content/wordpress-cron.md [hunk 1]
```diff
@@ -104,12 +104,10 @@ You can use external Crons if you want more control over your site's Cron jobs,
 
 ### Disable WP-Cron
 
-You will need to disable WP-Cron if your upstream does not have the definition below.
-
-Add the code below to your `wp-config.php` file to disable WP-Cron's internal processing:
+By default, Pantheon's WordPress upstreams have WP_CRON disabled. If you wish to continue using WP_CRON, you will need to add the code below to your `wp-config.php` file to enable WP-Cron's internal processing:
```
Comment (CdrMarks) [inline source/content/wordpress-cron.md:107]: @jspellman814 It'll be important to not the added line needs to be above the `require_once` of the `wp-config-pantheon.php` file.

**Review by CdrMarks — CHANGES_REQUESTED** [review]

**Review by CdrMarks — APPROVED** [review]

**Comment by CdrMarks** [comment]
[Preview](https://pr-7229--pantheon-docs.my.pantheonfrontend.website/docs/wordpress-cron)

File: source/content/wordpress-cron.md [hunk 2]
```diff
@@ -104,12 +104,10 @@ You can use external Crons if you want more control over your site's Cron jobs,
 
 ### Disable WP-Cron
 
-You will need to disable WP-Cron if your upstream does not have the definition below.
-
-Add the code below to your `wp-config.php` file to disable WP-Cron's internal processing:
+By default, Pantheon's WordPress upstreams have WP_CRON disabled. If you wish to continue using WP_CRON, you will need to add the code below to your `wp-config.php` file to enable WP-Cron's internal processing. This line needs to be above the `require_once` expression that pulls in `wp-config-pantheon.php`.
```
Comment (CdrMarks) [inline source/content/wordpress-cron.md:107]:
```suggestion
By default, Pantheon's WordPress upstreams have WP-CRON disabled. If you wish to continue using WP-CRON, you will need to add the code below to your `wp-config.php` file to enable WP-Cron's internal processing. This line needs to be above the `require_once` expression that pulls in `wp-config-pantheon.php`.
```

**Review by CdrMarks — CHANGES_REQUESTED** [review]

File: source/content/wordpress-cron.md [hunk 3]
```diff
@@ -104,12 +104,10 @@ You can use external Crons if you want more control over your site's Cron jobs,
 
 ### Disable WP-Cron
 
-You will need to disable WP-Cron if your upstream does not have the definition below.
-
-Add the code below to your `wp-config.php` file to disable WP-Cron's internal processing:
+By default, Pantheon's WordPress upstreams have WP-CRON disabled. If you wish to continue using WP-CRON, you will need to add the code below to your `wp-config.php` file to enable WP-Cron's internal processing. This line needs to be above the `require_once` expression that pulls in `wp-config-pantheon.php`.
```
Comment (CdrMarks) [inline source/content/wordpress-cron.md:107]:
```suggestion
By default, Pantheon's WordPress upstreams have WP-Cron disabled. If you wish to continue using WP-Cron, you will need to add the code below to your `wp-config.php` file to enable WP-Cron's internal processing. This line needs to be above the `require_once` expression that pulls in `wp-config-pantheon.php`.
```

**Review by CdrMarks — CHANGES_REQUESTED** [review]

**Review by CdrMarks — APPROVED** [review]

**Review by greg-1-anderson — APPROVED** [review]
Good to go once https://github.com/pantheon-systems/WordPress/pull/312 is merged.

**Comment by jspellman814** [comment]
@whitneymeredith @pantheondocs The next version of WordPress has been [merged](https://github.com/pantheon-systems/WordPress/pull/321), and the documented changes to WP-Cron are now in place. This PR is ready to merge ASAP.

**Pull request merged**